src/root-package-json.js

```diff
@@ -56,9 +56,9 @@ let scripts = {
       'lint:fix': 'npm run lint:fix --workspaces --if-present',
       prepare: 'npm run build',
       start: "concurrently 'npm:start:*' --restart-after 5000 --prefix-colors cyan,white,yellow",
-      'start:addon': `npm start --workspace ${addonName} -- --no-watch.clearScreen`,
-      'start:tests': `npm start --workspace ${testAppName}`,
-      test: `npm test --workspace ${testAppName}`,
+      'start:addon': `npm start --workspace ${addonName}`,
+      'start:test': `npm start --workspace ${testAppName}`,
+      test: 'npm run test --workspaces --if-present',
```

Comment:
I used `--workspaces --if-present` to better match the pattern for `lint` and `lint:fix` scripts.

Reply:
Ah, what I meant is, since the addon package does have a `test` script, we can run the root `test` script without the conditional check `--if-present`.